src/delphi/dataset/token_map.py

```diff
@@ -6,6 +6,15 @@
 def token_map(
     tokenized_dataset: Dataset,
 ) -> dict[int, list[tuple[int, int]]]:
+    """Return a mapping of tokens to their (prompt_idx, token_idx) locations in the tokenized_dataset.
+
+    Args:
+        tokenized_dataset (Dataset): A tokenized dataset.
+
+    Returns:
+        dict[int, list[tuple[int, int]]]: A mapping of tokens to their (prompt_idx, token_idx)
+            locations in the tokenized_dataset.
+    """
```

Comment:
first line of the docstring is enough, everything else is redundant

Reply:
Okay, will remove the other lines!